plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -195,6 +230,21 @@ func NewMaxPDVolumeCountPredicate(filter VolumeFilter, maxVolumes int, pvInfo Pe
 	return c.predicate
 }
 
+// getMaxVols checks the max PD volumes environment variable, otherwise returning a default value
+func getMaxVols(defaultVal int) int {
```

Comment:
This may not what we expected. The maximum volumes here can only be set by `os.env`. This is not the k8s way. I think we'd keep the old configurable way. @bsalamat WDYT?